spec/test_track_spec.rb

```diff
@@ -275,5 +277,25 @@
         end
       end
     end
+
+    context 'when set via full assets:precompile command' do
+      around do |example|
+        FileUtils.remove_dir('testtrack', true)
+        example.run
+        Rake::Task['app:assets:clobber'].invoke
```

Comment:
Should we hook into `assets:clobber` to remove the `testtrack/build_timestamp` file? Feels like that would match user expectations. (Maybe in a later PR so we don't introduce too much entropy here.)

Reply:
Yup, and we should **not** hook into `assets:clean` since it's common to run that directly after precompile to clear out older builds.